src/map/packet_system.cpp

```diff
@@ -2866,6 +2896,14 @@ void SmallPacket0x06E(map_session_data_t* session, CCharEntity* PChar, CBasicPac
         return;
     }
 
+    // check /blockaid
+    CCharEntity* PTarget = (CCharEntity*)PChar->GetEntity(targid, TYPE_PC);
```

Comment:
same thing here as earlier - don't do GetEntity if we don't need to.  we already get an entity further down from GetCharFromWorld (and it's worth noting that CCharEntity->GetEntity only gets an entity from this zone).  Additionally, blockaid will need to be checked on the receiving side, because an entity on another server will always fail this check (PTarget would be null, and PInvitee would also be null)